src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIdentifier.java

```diff
@@ -0,0 +1,102 @@
+/*
+UnnecessaryParentheses
+tokens = (default)EXPR, IDENT, NUM_DOUBLE, NUM_FLOAT, NUM_INT, NUM_LONG, \
+         STRING_LITERAL, LITERAL_NULL, LITERAL_FALSE, LITERAL_TRUE, ASSIGN, \
+         BAND_ASSIGN, BOR_ASSIGN, BSR_ASSIGN, BXOR_ASSIGN, DIV_ASSIGN, \
+         MINUS_ASSIGN, MOD_ASSIGN, PLUS_ASSIGN, SL_ASSIGN, SR_ASSIGN, STAR_ASSIGN, \
+         LAMBDA, TEXT_BLOCK_LITERAL_BEGIN, LAND, LITERAL_INSTANCEOF, GT, LT, GE, \
+         LE, EQUAL, NOT_EQUAL, UNARY_MINUS, UNARY_PLUS, INC, DEC, LNOT, BNOT, \
+         POST_INC, POST_DEC
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessaryparentheses;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class InputUnnecessaryParenthesesIdentifier {
+    public static boolean is(Class<?> clazz) {
+        return true
+            && (test(clazz)); // violation 'Unnecessary parentheses around identifier 'test'.'
+    }
+
+    public static boolean isOk(Class<?> clazz) {
+        return true
+            && test(clazz); // ok
+    }
+
+    public int square(int a, int b){
+    int square = (a * b); // violation 'Unnecessary parentheses around assignment right-hand side'
+    return (square);      // violation 'Unnecessary parentheses around identifier 'square''
+    }
+
+    public int square1(int a, int b){
+    int square = a * b; // ok
+    return square;      // ok
+    }
+
+    public static boolean test(Class<?> clazz) {
+        return true;
+    }
+
+    public static boolean newTest(Class<?> clazz) {
+        return true
+            && test((clazz)); // violation 'Unnecessary parentheses around identifier 'clazz''
+    }
+
+    public static boolean newTest1(Class<?> clazz) {
+        return true
+            && test(clazz); // ok
+    }
+
+    public static boolean newTest2(Class<?> clazz) {
+        return true
+            && ((test((clazz)))); // 2 violations
+    }
+
+    public String getMarkerNumber() {
+        String markerNumber = "someText";
+        return markerNumber;
+    }
+
+    public boolean m3() {
+        return true;
+    }
+
+    public void testIF() {
+        int a = 9;
+        int b = 10;
+        if (a > b) {
+            System.out.println("a is greater then b");
+        }
+        else if ((b < a) && ((8>9) || (!m3()))){ // 3 violations
```

Comment:
Unless there is a reason, please split this into multiple lines to show the violation messages.

Reply:
@rnveach  should i split all the code in which their are multiple violation

Reply:
this change has been 
done

Reply:
If you can do it without harming the tests, then yes.

Reply:
done

Reply:
Thanks